src/components/AffectedOfferings.vue

```diff
@@ -50,6 +50,10 @@ const isAnythingExpanded = computed(() => (
   || Object.values(expandedModules.value).some(Boolean)
 ));
 
+const isAnythingCollapsed = computed(() => (
+  !Object.values(expandedModules.value).every(Boolean)
```

Comment:
This is fine - I might have used Object.values(expandedModules.value).some(Boolean)